src/custom/pages/Rewards/index.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import Page, { Title, Content, GdocsListStyle } from 'components/Page'
+import styled from 'styled-components'
+import { Trans } from '@lingui/macro'
+import CowsImg from 'assets/images/cows.png'
```

Comment:
Move the image to the custom folder, such as https://github.com/gnosis/cowswap/blob/develop/src/custom/assets/cow-swap/cow-load.gif

Also, I suggest a more descriptive name, as there are other `cows` images.
Something like, `cows-side-by-side`